com.microsoft.mrtk.uxcore/Button/RectTransformColliderFitter.cs

```diff
@@ -74,6 +78,8 @@ public Vector2 Padding
 
         private ScrollRect scrollRect;
 
+        private bool maskApplied = false;
```

Comment:
Could you please move this to be a private return var inside your UpdateMask function?